### PR TITLE
removes docs and fixed build_docs.sh to only build xml for doxygen

### PR DESCRIPTION
**Pull request by acostadon**
resolves issue https://github.com/rapidsai/cugraph-docs/issues/63

### CODE REVIEW

File: ci/build_docs.sh [hunk 1]
```diff
@@ -52,13 +52,6 @@ rapids-mamba-retry install \
 
 export RAPIDS_DOCS_DIR="$(mktemp -d)"
 
-# for PROJECT in libwholegraph; do
-#   rapids-logger "Download ${PROJECT} xml_tar"
-#   TMP_DIR=$(mktemp -d)
-#   export XML_DIR_${PROJECT^^}="$TMP_DIR"
-#   curl "https://d1664dvumjb44w.cloudfront.net/${PROJECT}/xml_tar/${RAPIDS_VERSION_NUMBER}/xml.tar.gz" | tar -xzf - -C "${TMP_DIR}"
-# done
```
Comment (jameslamb) [inline ci/build_docs.sh:60]: Ah great, thanks for removing this too!

**Review by jameslamb — APPROVED** [review]
Excellent! Now that https://github.com/rapidsai/cugraph-docs/pull/65 is merged, I think this is safe to merge as soon as we see a successful build on `branch-25.02` there.

Here's the build to watch: https://github.com/rapidsai/cugraph-docs/actions/runs/12398664118/job/34611886692

**Review by BradReesWork — APPROVED** [review]

**Comment by jameslamb** [comment]
Merging based on offline conversation with @acostadon . We are close to having docs-build set up from https://github.com/rapidsai/cugraph-docs, and preventing docs builds from this repo will help there.

**Pull request merged**